js/sites.js

```diff
@@ -97,7 +99,7 @@
      * @return {Boolean} Whether this site is pinned.
      */
     isPinned() {
-      return this._link.pinState;
+      return gPinnedLinks.isPinned(this._link);
```

Comment:
I believe you added in `pinState` since the state was kept in the parent and sent back and forth to the via msgs. Is that correct? The pin state is now accessible in content via indexedDB. Alternatively we can keep the `pinState` attribute and not need `isPinned()`